src/TurbulenceConvection/EDMF_Precipitation.jl

```diff
@@ -16,8 +134,7 @@ function compute_precipitation_sink_tendencies(
 
     nothing
 end
-
-function compute_precipitation_sink_tendencies(
+function grid_mean_precipitation_sink_tendencies(
```

Comment:
@szy21 - this is a grid-mean tendency from grid-mean precipitation sinks. We could move it somewhere into ClimaAtmos?

Reply:
Maybe move it to src/parameterizations/microphysics/precipitation.jl?